src/did/web5-did.js

```diff
@@ -174,13 +172,4 @@ class Web5DID {
     if (!api) throw `Unsupported cryptographic cipher: ${name}`;
     return api;
   }
-
-  /**
-   * Utility functions for working with DIDs
-   */
-  #util = { ...DidUtils };
 }
-
-export {
-  Web5DID,
-};
```

Comment:
why was this changed?

Reply:
Use of inline exports and separate exports was inconsistent throughout Web5 JS. Adopted the style used throughout `dwn-sdk-js`.  More work needed to update the rest of Web5 JS but this is purely a style difference.

Reply:
this was consistent before ca736832e9c3a8b777f7784def4a5f36a27928ca :/

Reply:
Inconsistency addressed in 1b4d64ec4f93fbcd7785ff2f99dafc6c7c7c6750